### DISCUSSION TITLE
Allow hiding/showing panels based on selected date range/variable

### DISCUSSION THREAD

**Discussion by brianmario** [grafana/grafana#37848] [Dashboards, Panels & Visualizations]
Hello!

After looking through the list of existing issues, it looks like #18356 is probably the closest thing to what I'm trying to achieve. Though based on the description, it isn't quite the use-case I'm going for (though I'm sure if the functionality existing to allow for it, my use-case would be doable as well). For the rest of the related issues, #4470 seems to be the tracking issue for pretty much the rest of them.

My use-case is with farming (vineyard). There are various stats that are useful during certain times of the growing season, but then aren't useful after certain events happen out in the vineyard. One of these is rainfall, which for California essentially stops and will be a zero-data graph from late-Spring until Winter. I saw some issues related to hiding graphs that don't have data - like #6684 - which would help for the rainfall graph, but there are other graphs that continue to have data which don't need to be shown anymore based on the date. One of those is for tracking "powdery mildew degrees" - an accumulation of degrees during each day (generally between 70-85 F) which contribute to initial mildew growth, which in turn inform us as to when to perform our first application of fungicide. The values in this graph will continue to accumulate all year, but are no longer useful to us after we've applied our first application/spray.

It sounds like that can potentially be driven by use of variables in Grafana, but it looks like the functionality to hide a panel based on that doesn't exist yet?

Apologies ahead of time if this functionality _does_ exist. I'm not new to Grafana, but I'm pretty far from expert-level on how it all works 😉 

I know I could probably create different dashboards for this (one for pre-season graphs, one for mid-season and maybe even one for post-season) but I'd like to have this data on display via a kiosk in our shop in such a way that it doesn't require _too_ much interaction from less computer savvy folks. A more dynamic dashboard driven by me would definitely be ideal.

Thank you for your work on this amazingly useful project!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
